spec/models/family_spec.rb

```diff
@@ -38,4 +38,11 @@
   it "when a family is created the first authorized family member is created" do
     expect { create(:family) }.to change { AuthorizedFamilyMember.count }.by(1)
   end
+
+  describe "export_json" do
+    it "returns a hash" do
+      family = build(:family)
+      expect(family.export_json).to be_a(Hash)
+    end
+  end
```

Comment:
We may want to change the name of the method or the output. The method's name is `export_json` but is expected to return a `Hash`. Maybe `export_hash`?